client/src/modules/dashboard/Chapters/pages/ChapterPage.tsx

```diff
@@ -133,7 +188,19 @@ export const ChapterPage: NextPageWithLayout = () => {
               {loadingCalendar ? (
                 <Spinner size="sm" />
               ) : data.dashboardChapter.calendar_id ? (
-                <CheckIcon boxSize="5" />
+                <>
+                  <CheckIcon boxSize="5" />
+                  {checkPermission(user, Permission.ChapterEdit, {
```

Comment:
```suggestion
                  {checkPermission(user, Permission.ChapterCreate, {
```
This is fairly serious and (hopefully) not something people will often need, so I think it should be restricted to instance owners.